node/src/rpc/rpc_effects.rs

```diff
@@ -500,23 +494,14 @@ pub fn rpc_effects<S: Service>(store: &mut Store<S>, action: RpcActionWithMeta)
                 ),
                 Err(err) => RpcSnarkerJobSpecResponse::Err(err),
             };
-            if store
```

Comment:
wondering if this wasn't a placeholder for some error handling/reporting. Lets add a `// TODO: handle potential errors` or something like that here and bellow on top of `let _ = ...`

Reply:
I wasn't sure about that, but there are a lot of bindings like that for `Result`.

Reply:
Yes, there are a a bunch of ignored errors (mostly unlikely ones or not very important ones, but it is not that good to silently ignore them). At some point we will need to handle all those (at least some logging so that such errors are not invisible).